Deploy/kubernetes/dockerfiles/ui.dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.6.2
```

Comment:
A general note: you should separate building and packaging to two different dockerfiles; so you dont end up with large images.